File: sort-colors/sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        colors = [0,0,0]
        for n in nums:
            colors[n] += 1
        nums.clear()
        for color in range(3):
            for _ in  range(colors[color]):
                nums.append(color)

File: sort-colors/sort-colors.c
void sortColors(int* nums, int numsSize){
    int colors[]={0,0,0};
    for(int i=0;i<numsSize;i++){
        colors[nums[i]]++;
    }
    int i=0;
    for(;i<colors[0];i++)nums[i]=0;
    for(;i<colors[0]+colors[1];i++)nums[i]=1;
    for(;i<numsSize;i++)nums[i]=2;
}
